### PR TITLE
Feedback wcag 2 0

### PR DESCRIPTION
**Pull request by Katrienvh**
## Type

- Enhancement

## Pull request description

Fix some accessibility issues in the markup

### CODE REVIEW

File: src/Backend/Modules/Pages/Layout/Templates/Edit.html.twig [hunk 1]
```diff
@@ -106,7 +106,7 @@
               {# Do not change the ID! #}
               <p class="current-template">{{ 'lbl.Template'|trans|ucfirst }}:
                 <span id="tabTemplateLabel">&nbsp;</span>
-                <a data-toggle="modal" data-target="#changeTemplate">
+                <a href="#" data-toggle="modal" data-target="#changeTemplate">
```
Comment (mathiashelin) [inline src/Backend/Modules/Pages/Layout/Templates/Edit.html.twig]: I think this should be a button-element.

File: src/Backend/Modules/Pages/Layout/Templates/Add.html.twig [hunk 2]
```diff
@@ -73,7 +73,7 @@
               {# Do not change the ID! #}
               <p class="current-template">{{ 'lbl.Template'|trans|ucfirst }}:
                 <span id="tabTemplateLabel">&nbsp;</span>
-                <a data-toggle="modal" data-target="#changeTemplate">
+                <a href="#" data-toggle="modal" data-target="#changeTemplate">
```
Comment (mathiashelin) [inline src/Backend/Modules/Pages/Layout/Templates/Add.html.twig]: I think this should be a button-element.

Comment (carakas) [inline src/Backend/Modules/Pages/Layout/Templates/Add.html.twig]: shouldn't this be a button ?

File: src/Backend/Modules/Pages/Layout/Templates/Edit.html.twig (hunk 1 above)
Comment (carakas) [inline src/Backend/Modules/Pages/Layout/Templates/Edit.html.twig]: shouldn't this be a button ?

File: src/Backend/Modules/Pages/Layout/Templates/Add.html.twig (hunk 2 above)
Comment (Katrienvh) [reply src/Backend/Modules/Pages/Layout/Templates/Add.html.twig]: screen shot 2017-11-08 at 11 02 23
it isn't a button

File: src/Backend/Modules/Pages/Layout/Templates/Edit.html.twig (hunk 1 above)
Comment (Katrienvh) [reply src/Backend/Modules/Pages/Layout/Templates/Edit.html.twig]: screen shot 2017-11-08 at 11 02 23
it isn't a button

File: src/Backend/Modules/Pages/Layout/Templates/Add.html.twig (hunk 2 above)
Comment (Katrienvh) [reply src/Backend/Modules/Pages/Layout/Templates/Add.html.twig]: screen shot 2017-11-08 at 11 02 23
it isn't a button

File: src/Backend/Modules/Pages/Layout/Templates/Edit.html.twig (hunk 1 above)
Comment (Katrienvh) [reply src/Backend/Modules/Pages/Layout/Templates/Edit.html.twig]: screen shot 2017-11-08 at 11 02 23
it isn't a button

Comment (carakas) [reply src/Backend/Modules/Pages/Layout/Templates/Edit.html.twig]: but it works like a button, didn't they say that a link always needed to go to a page or resource?

File: src/Backend/Modules/Pages/Layout/Templates/Add.html.twig (hunk 2 above)
Comment (carakas) [reply src/Backend/Modules/Pages/Layout/Templates/Add.html.twig]: but it works like a button, didn't they say that a link always needed to go to a page or resource?

File: src/Backend/Modules/Pages/Layout/Templates/Edit.html.twig (hunk 1 above)
Comment (carakas) [reply src/Backend/Modules/Pages/Layout/Templates/Edit.html.twig]: but it works like a button, didn't they say that a link always needed to go to a page or resource?

File: src/Backend/Modules/Pages/Layout/Templates/Add.html.twig (hunk 2 above)
Comment (carakas) [reply src/Backend/Modules/Pages/Layout/Templates/Add.html.twig]: but it works like a button, didn't they say that a link always needed to go to a page or resource?

File: src/Backend/Modules/Pages/Layout/Templates/Edit.html.twig (hunk 1 above)
Comment (mathiashelin) [reply src/Backend/Modules/Pages/Layout/Templates/Edit.html.twig]: Maybe adding `role="button"` is the best solution here?

File: src/Backend/Modules/Pages/Layout/Templates/Add.html.twig (hunk 2 above)
Comment (mathiashelin) [reply src/Backend/Modules/Pages/Layout/Templates/Add.html.twig]: Maybe adding `role="button"` is the best solution here?

**Pull request merged**